### PR TITLE
Add documentation for Beaker Interactive

### PR DESCRIPTION
**Pull request by aaasen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ rapid, reproducible research.
 * [Install](/docs/start/install.md)
 * [Your First Experiment](/docs/start/experiment.md)
 * [Image and dataset](/docs/start/image.md)
+* [Beaker Interactive](/docs/start/interactive.md)
```
Comment (ckarenz) [inline README.md:17]: "Interactive Sessions"?

File: docs/start/interactive.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Beaker Interactive is a way to use Beaker to quickly experiment
+without the overhead of creating an image each time you change your code.
+
+## Select a Machine
+
+Beaker Interactive is available on any machine that is running Beaker.
+You can see a list of available machines by first selecting a cluster
+from the [list of on-premise clusters](https://beaker.org/clusters).
+Once you have a cluster, select a node within that cluster and get its hostname.
```
Comment (ckarenz) [inline docs/start/interactive.md:12]:
```suggestion
Once you have a chosen cluster, select any node to find its hostname.
```

File: docs/start/interactive.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Once on the machine, run `beaker session create` to start a new session.
+This will ask for your user token which is used to authenticate with Beaker.
+Beaker will then claim resources on the node, which may take a while if the node is full.
+Finally, Beaker will create an interactive session for you.
+
+## Interactive Environment
+
+Each interactive session is backed by a Docker container.
+Within the container, you are free to do whatever you want; you cannot affect other processes on the machine.
```
Comment (ckarenz) [inline docs/start/interactive.md:29]:
```suggestion
Each interactive session hosted in a container sandbox.  Within the container
you are free to run or install whatever you want; you cannot affect other processes on the machine.
```

File: docs/start/interactive.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+Everything except for your home directory and `/net` is destroyed once the session stops.
+
+By default, interactive sessions use the `allenai/base:cuda11.2-ubuntu20.04` Docker image which is maintained by the Beaker team.
+This image is based on Ubuntu 20.04 and already has CUDA 11.2 drivers installed.
+It also comes with some useful tools like the AWS CLI, Google Cloud CLI, and the Beaker CLI.
+If you find that any tools are missing, please contact the Beaker team and we will consider adding them.
```
Comment (ckarenz) [inline docs/start/interactive.md:35]:
```suggestion
If you think we've missed a common tool, please contact the team and we will consider adding them.
```

File: docs/start/interactive.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Your home directory is mounted into the container so anything you write to `~` will persist between sessions.
+NFS is also available in the `/net` directory.
```
Comment (ckarenz) [inline docs/start/interactive.md:38]:
```suggestion
Your home directory is mounted into the container so anything you write to `~` will persist
between sessions on that node. NFS is also available in the `/net` directory.
```

File: docs/start/interactive.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+### Additional GPUs
+
+By default, sessions are assigned 1 GPU.
```
Comment (ckarenz) [inline docs/start/interactive.md:42]: We should consider whether we do this.  0 might be a more predictable default.  See resource negotiation issue linked in Slack.

File: docs/start/interactive.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+For example, the command below prints the Python version.
+You could also use this to run a Python script in your home directory.
+
+```
+beaker session create -- python3 --version
+```
+
+### Alternative Base Images
+
+If you need to use a different base image, pass the `--image` flag when creating a session.
```
Comment (ckarenz) [inline docs/start/interactive.md:64]: How does user account mapping work in this environment?  Should we put in a caveat that the user name will look weird?

**Review by ckarenz — APPROVED** [review]
Looks good!  Comments are optional but recommended.  Having any documentation at all is an improvement.

File: docs/start/interactive.md (hunk 7 above)
Comment (aaasen) [reply docs/start/interactive.md:64]: You're right, it doesn't quite work. I'll add a caveat.

File: README.md (hunk 1 above)
Comment (aaasen) [reply README.md]: I like "interactive sessions". Updated.

**Pull request merged**